os_window_x11.c

```diff
@@ -94,14 +110,16 @@ os_window_create(uint32_t width, uint32_t height) {
   Visual* p_visual;
   int depth;
   size_t map_size;
-  struct rm_window_shmid_error_handler_context_struct error_handler_context;
+  XErrorHandler old_error_handler;
 
   p_window = malloc(sizeof(struct os_window_struct));
```

Comment:
Apologies; you're likely to get a merge conflict here because I replaced this verbose sequence with util_mallocz() on master. Just an FYI.